src/plugins/discourse/createGraph.js

```diff
@@ -131,7 +142,13 @@ export function _createGraphData(
     const createsLike = NE.createsLikeEdge(serverUrl, like);
     const likes = NE.likesEdge(serverUrl, like);
     const user = repo.findUser(like.username);
-    const weight = likeWeight(user);
+    const post = repo.postById(like.postId) || {};
```

Comment:
So if we were to type this out we would write `const post: Post | {} = ...`. That feels much less natural than `const post: ?Post` or `const post: Post | null`. Fix?